src/psycopmlutils/synth_data_generator/synth_txt_data_generator.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
```

Comment:
Add a document docstring stating what the script does (especially when there is a if __name__ ...

Reply:
can be very short